include/service.h

```diff
@@ -45,7 +45,7 @@ class Service {
                             std::vector<discovered_alleles>& ans);
 
     /// Genotype a set of samples at the given sites, producing a BCF file.
-    Status genotype_sites(const genotyper_config& cfg, const std::string& sampleset, const std::vector<unified_site>& sites, const std::string& filename, consolidated_loss& dlosses);
+    Status genotype_sites(const genotyper_config& cfg, const std::string& sampleset, const std::vector<unified_site>& sites, const std::string& filename, consolidated_loss& dlosses, bool write_vcf=false);
```

Comment:
Let's make the output format an entry in `genotyper_config`. I envision we'll eventually have additional possible output formats, such as CSV to facilitate loading into a RDBMS.